helm/cert-manager-app/templates/startupapicheck-psp-clusterrole.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.startupapicheck.enabled -}}
+kind: ClusterRole
```

Comment:
Any reason to not picking up `{{- if .Values.global.podSecurityPolicy.enabled }}` from [upstream](https://github.com/jetstack/cert-manager/blob/master/deploy/charts/cert-manager/templates/startupapicheck-psp-clusterrole.yaml)? Just would be nice to follow upstream as close as possible.

Reply:
While following upstream as close as possible is a very good thing, PSP is required in GS clusters anyway, so trying to keep configuration as simple as possible

Reply:
oki